tests/dummy_qrc/qblox.yml

```diff
@@ -15,21 +15,13 @@ instruments:
       power: 4
 
     qcm_bb0:
-        o1:
-            offset: 0.1
-
-        o2:
-            offset: 0.11
-
-        o3:
-            offset: 0.12
-
-        o4:
-            offset: 0.13
+        o1: True
+        o2: True
+        o3: True
+        o4: True
```

Comment:
If there is no offset, do we need this at all? Or could we make it a list instead?

Reply:
As written also here https://github.com/qiboteam/qibolab_platforms_qrc/pull/87#issuecomment-1838106964 ideally no. 

I think the solution for this is to create the ports dynamically when the user asks for them in the create method, doing:
```py
channels |= Channel(name="L4-5", port=modules["qcm_bb0"].ports["o1"])
```
If a port is not asked for there, then we don't create it at all. This should replace the `True`/`False` and we can drop this section completely.

I am not sure how easy is to implement this, though.